Documentation/Storage-Configuration/Advanced/ceph-osd-mgmt.md

```diff
@@ -190,3 +190,27 @@ If you don't see a new OSD automatically created, restart the operator (by delet
 
 !!! note
     The OSD might have a different ID than the previous OSD that was replaced.
+
+
+## OSD Encryption as Day-2 operation
+
+Existing PVC based OSDs can be migrated to enable or disable encryption as a day-2 operation. To start migration of the existing OSDs to enable or disable encryption: 
```

Comment:
There are going to be multiple scenarios where OSDs can be migrated. How about we phrase this so that we can easily add to the list of supported scenarios? Something like the following, and moving some of your bulleted text below up to this intro section.
```
Ceph does not support changing certain settings on existing OSDs. 
To support changing these settings on an OSD, the OSD must be destroyed and re-created with the new settings. 

Rook will automate the update by migrating only one OSD is migrated at a time. The operator waits for the data to rebalance (pgs to become `active+clean`) before migrating the next OSD. This ensures that there is no data loss.